BootloaderCorePkg/BootloaderCorePkg.dec

```diff
@@ -111,6 +111,9 @@
 
   gPlatformModuleTokenSpaceGuid.PcdSmbiosTablesSize       |     0x1000 | UINT16 | 0x200000E3
 
+  # Size of the Hash store allocated in bootloader
+  gPlatformModuleTokenSpaceGuid.PcdHashStoreSize       | 0x00000000 | UINT32 | 0x200000F1

```

Comment:
Change the default to be  0x200  , maybe ?

Reply:
Will do